A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -558,6 +558,7 @@ class CDUPerformancePage {
         mcdu.onLeftInput[2] = (value) => {
             if (mcdu.setPerfApprWind(value)) {
                 mcdu.updateTowerHeadwind();
+                mcdu.updateApproachSpeeds();
```

Comment:
We update the speeds immediately when appr wind or config is changed so the UI can actually update immediately. Otherwise the UI updates but re-uses the old speeds because the periodic task that updates mcdu.approachSpeeds won't have had a chance to update them, and then the approach page doesn't update again for some time.